simulation_parameters/microbe_stage/biomes.json

```diff
@@ -46,19 +46,19 @@
           "Name": "SMALL_IRON_CHUNK",
           "Meshes": [
             {
-              "ScenePath": "res://assets/models/IronRock1.tscn",
+              "VisualResource": 9,
```

Comment:
Yeah this, is not very nice because people need to manually write this file so using string keys here would be vastly superior.

When referring to the integer values I meant how this will be saved in the game saves, which are not edited by humans so the compactness of just pure numbers helps there.

Reply:
So what do I put here instead if visual id is `BigIronChunk1`?

Reply:
`"BigIronChunk1"`
which either works immediately or a custom JSON converter is needed that supports both string and integer values.

Reply:
JSON converter is needed then. Unfortunately.

Reply:
I just looked and so far it seems like no enum values have been wanted to read from strings in the game. The RevolutionaryWebApp repo has that setup already, but sadly it uses a different JSON lib, so a separate approach is needed here.